app/(auth)/welcome/index.tsx

```diff
@@ -1,11 +1,53 @@
-import { Link } from 'expo-router'
-import { Text, View } from 'react-native'
+import { useState } from 'react'
+import { View } from 'react-native'
+import PagerView from 'react-native-pager-view'
+import { ButtonLink, CarouselIndicators, Welcome } from '../../../components'
+
+const url = '../../../assets/images/'
+
+const imageSource = {
+  primarySource: require(`${url}welcome_background.webp`),
+  secondarySource: require(`${url}welcome_background_3.webp`),
+  thirdSource: require(`${url}welcome_background_2.webp`)
+}
+
+const carouselSection = [
+  {
+    key: 0,
+    title: 'Welcome to Mova',
+    detail: 'The best movie streaming app of the century to make your days great',
+    url: imageSource.primarySource
+  },
+  {
+    key: 1,
+    title: 'The best movies',
+    detail: '¡you can watch the best movies when you wish it!',
+    url: imageSource.secondarySource
+  },
+  {
+    key: 2,
+    title: 'The best series',
+    detail: 'you can watch the best series when you wish it',
+    url: imageSource.thirdSource
+  }
+]
 
 function WelcomeScreen() {
+  const [getDataPager, setGetDataPager] = useState(0)
```

Comment:
cambiar a un nombre mas descriptivo, en su lugar : 
```suggestion
  const [viewPosition, setViewPosition] = useState(0)
```